src/openforms/submissions/tasks/co_sign.py

```diff
@@ -30,17 +32,14 @@ def send_email_cosigner(submission_id: int) -> None:
         )
         return
 
-    # Send Co-sign email # TODO add configurable templates
-    content = render_email_template(
-        ugettext(
-            'This is a request to co-sign form "{{ form_name }}". Please go to the webpage of this form on our '
-            "website and click on the 'co-sign' button. You will then be redirected to authenticate yourself."
-            "After authentication, fill in the following code to retrieve the form submission:\n\n{{ code }}\n\n."
-        ),
-        context={
+    template = get_template("emails/cosign.html")
+    content = template.render(
+        {
             "code": submission.public_registration_reference,
             "form_name": submission.form.name,
-        },
+            "form_url": submission.form.get_absolute_url(),
```

Comment:
```suggestion
            "form_url": submission.form_url,
```
This needs to be the link to the form on the frontend